docker-compose.yml

```diff
@@ -193,16 +199,19 @@ services:
       - DD_APM_ENABLED=true
       - DD_APM_NON_LOCAL_TRAFFIC=true
       - DD_LOGS_ENABLED=true
+      - DD_HOSTNAME=172.43.0.4
```

Comment:
This needs to be set to make sure we are able to send logs into the agent since by default it uses localhost